src/components/DurationDropdown/DurationDropdown.tsx

```diff
@@ -36,9 +37,10 @@ export class DurationDropdown extends React.Component<DurationDropdownProps> {
         disabled={this.props.disabled}
         handleSelect={key => this.props.setDuration(Number(key))}
         value={String(this.props.duration)}
-        label={this.props.durations[this.props.duration]}
-        options={this.props.durations}
+        label={serverConfig.durations[this.props.duration]}
```

Comment:
it seems `this.props.durations` isn't used anywhere now, or did I miss one? So it could be removed from props

Reply:
Seems unused, I'll remove it.